Documentation/compatibility/wpf-accessibility-improvements.MD

```diff
@@ -0,0 +1,101 @@
+## 195: Accessibility improvements in WPF
+
+### Scope
+Major
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+__High Contrast improvements__
+
+- Focus for expander was not visible.  This is now fixed.
+- The text in CheckBox and RadioButton is difficult to see when selected.  This issue is now fixed.
+- The border of a disabled ComboBox is now the same color as the disabled text.
+- Disabled and focused buttons used the incorrect theme color.  This is now fixed.
+- Setting a ComboBox’s style to Toolbar.ComboBoxStyleKey caused the dropdown button to be invisible.  This is now fixed.
+- The sort indicator arrow in DataGrid used the incorrect theme colors, making it confusing.  This is now fixed.
+- The default hyperlink style changed to incorrect color on mouse over in high contrast themes.  This is now fixed.
+- Keyboard focus on radio buttons in high contrast themes is not visible.  This is now fixed.
+- DataGrid checkbox column does not use the expected colors for keyboard focus feedback.  This is now fixed.
+- Keyboard focus visuals are not visible on ComboBoxes and ListBoxes in high constrast themes.  This is now fixed.
+
+
+__Screen reader interaction improvements__
+
+- Expanders were announced by screen readers as buttons, they are now correctly announced as groups (expand/collapse).
+- DataGridCells were announced by screen readers as “custom”, they are now correctly announced as data grid cell (localized)
+- Screen readers wouldn’t announce the name of an editable ComboBox, this issue is now fixed.
+- PasswordBoxes were announced as “no item in view” or had otherwise incorrect behavior, this issue is now fixed.
+
+
+__LiveRegion support__
+
+Screen readers such as Narrator help people know the UI contents of an application, usually by describing something about the UI that’s currently focused, as that is probably the element of most interest to the user at any given moment. However, if a UI element changes somewhere in the screen and it is not being focused at that point in time, the user may not be informed, and so they may be missing important information.
```

Comment:
as that --> because that
delete "at any given moment"
"it is not being focused at that point in time" --> "it does not have the focus"
may not be informed, and so they may be missing --> may not be informed and miss